docs/EN_US/DataPatterns/DataPatternsMods/DataPatternsInECLWatch.xml

```diff
@@ -1,18 +1,189 @@
+      Files Details page.</para>
+
+      <graphic fileref="../../images/DP005.jpg"/>
+
+      <para><emphasis role="bold">Note</emphasis>: Key files cannot be used
+      with Data Patterns.</para>
+    </listitem>
+
+    <listitem>
+      <para>The first time you open the Data Patterns tab, a notification
+      tells you that an analysis is not available. Press the Analyze button to
+      generate a Profile analysis. Once complete, the analysis
+      displays.</para>
+
+      <graphic fileref="../../images/DP003.jpg"/>
+    </listitem>
+
+    <listitem>
+      <para>Click on the title “Data Patterns” on the bar located near the top
+      of the page. Clicking on the Data Patterns title will generate a report
+      of the data file.</para>
+    </listitem>
+  </orderedlist>
+
+  <para>Several columns organize each row of information, and each row of
+  information represents every attribute in the Profile result.</para>
+
+  <para>Here is a short description of each column:</para>
+
+  <itemizedlist>
+    <listitem>
+      <para>Type information, Cardinality Count &amp; Filled Count.</para>
+    </listitem>
+
+    <listitem>
+      <para>Min, Avg, Max Length (for string attributes) or Mean, Std.
+      Deviation, Quartiles (for numeric attributes).</para>
+    </listitem>
+
+    <listitem>
+      <para>Quartile bell curve and candlestick</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>only shown for numeric attributes.</para>
+        </listitem>
+
+        <listitem>
+          <para>this column is not displayed if there are no numeric
+          attributes.</para>
+        </listitem>
+      </itemizedlist>
+    </listitem>
+
+    <listitem>
+      <para>Cardinality Breakdown listed by count descending</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>only shown for attributes with cardinality_breakdown
+          content.</para>
+        </listitem>
+
+        <listitem>
+          <para>this column is not displayed if there are no numeric
+          attributes.</para>
+        </listitem>
+      </itemizedlist>
+    </listitem>
+
+    <listitem>
+      <para>Popular Patterns</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>only shown for attributes with popular_patterns
+          content.</para>
+        </listitem>
+
+        <listitem>
+          <para>this column is not displayed if there are no numeric
+          attributes.</para>
+        </listitem>
+      </itemizedlist>
+    </listitem>
+  </itemizedlist>
+
+  <para>For example:</para>
+
+  <graphic fileref="../../images/DP004.jpg"/>
+
+  <para>As seen in the “state” section, the cardinality of each state is
+  displayed. The cardinality breakdown is shown for attributes with a low
+  number of unique, non-nil values, showing each value and the number of
+  records that contain the value.</para>
+
+  <para><emphasis role="bold">Note</emphasis>: The lcbLimit parameter governs
```

Comment:
Maybe include a link to the programmatic invocation of Profile() for additional info on what lcbLimit means?